src/content-handlers/iiif/JQueryPlugins.ts

```diff
@@ -558,11 +558,13 @@ interface JQuery {
   css: any;
   append: any;
   text: any;
+  link: any;
```

Comment:
I don't doubt that these type adjustments are here for a reason, but I can't figure out what that reason is, since I don't see references to .link or .render in GalleryComponent.ts. What am I missing? Is it possible that these are relics from before we removed JSRender from the gallery component?